canbench-bin/src/main.rs

```diff
@@ -21,6 +22,27 @@ struct Args {
     less_verbose: bool,
 }
 
+#[derive(Debug, Deserialize)]
+struct InitArgs {
+    // hex encoded argument to pass to the canister
+    hex: String,
+}
+
+#[derive(Debug, Deserialize)]
+struct Config {
+    // If provided, instructs canbench to build the canister
+    build_cmd: Option<String>,
+
+    // Where to find the wasm to be benchmarked
+    wasm_path: String,
+
+    // If provided, instructs canbench to store the results in this file
```

Comment:
```suggestion
    // If provided, instructs canbench to store the results in this file
    // Otherwise, `canbench_results.yml` is used by default
```